lib/models/package_wrapper.freezed.dart

```diff
@@ -28,7 +28,7 @@ class _$PackageTearOff {
       @JsonKey(name: 'packageType', unknownEnumValue: PackageType.unknown)
           PackageType packageType,
       @JsonKey(name: 'product')
-          Product product,
+          StoreProduct product,
```

Comment:
this is an interesting one. I checked and in Android we still have `product` but in `iOS` we have `storeProduct`. I would be opposed to change it to reduce the amount of breaking changes but we should be consistent. So maybe we should do `storeProduct` since iOS is the more popular one

Reply:
ok so I changed it to storeProduct but keeping the key in the dictionary as product, since that's what iOS is doing right now. If it gets updated, I will update it